pkg/apis/servicecatalog/v1alpha1/defaults_test.go

```diff
@@ -68,6 +68,22 @@ func roundTrip(t *testing.T, obj runtime.Object) runtime.Object {
 	return obj3
 }
 
+func TestSetDefaultServiceBroker(t *testing.T) {
```

Comment:
Tests here need to change according to the logic as I commented above

Reply:
There should be test cases for:

1.  Neither duration or behavior set
2.  Behavior set to manual
3.  Behavior set to duration but no duration provided

Reply:
Still need test cases for these defaults.  You need to ensure that after an object starting in each of the states in the above comment is defaults, that the object is in the expected state.